### PR TITLE
Adds sort feature.

### PR DESCRIPTION
**Pull request by elcct**

### CODE REVIEW

File: datahub/search/elasticsearch.py [hunk 1]
```diff
@@ -46,7 +46,28 @@ def get_search_term_query(term):
...
+    }
+    return name_map.get(field, field)
+
+
+def get_sort_query(qs, field_order=None):
+    """Attaches sort query."""
+    if field_order is None:
+        return qs
+
+    tokens = field_order.split(':')
```
Comment (reupen) [inline datahub/search/elasticsearch.py]: I think str.rparitition() or str.rsplit(maxsplit=1) would be better here.

File: datahub/search/views.py [hunk 2]
```diff
@@ -145,13 +151,15 @@ def post(self, request, format=None):
...
         limit = int(request.data.get('limit', 100))
 
         results = elasticsearch.get_search_investment_project_query(
             term='',
             filters=filters,
             ranges=ranges,
+            field_order=sortby,
             offset=offset,
             limit=limit,
         ).execute()
```
Comment (reupen) [inline datahub/search/views.py]: Should check the response status code in these tests.

File: datahub/search/elasticsearch.py (hunk 1 above)
Comment (elcct) [inline datahub/search/elasticsearch.py]: Good idea. Added rsplit.

File: datahub/search/views.py (hunk 2 above)
Comment (elcct) [inline datahub/search/views.py]: Indeed, somehow missed that. Thanks!

**Comment by elcct** [comment]
@reupen we have not defined which fields we want to sort by, writing validation for all fields seems disproportionate. I was thinking about adding validation once we get better idea what is really needed.

**Comment by reupen** [comment]
It shouldn't blow up if you pass an invalid field. It needs to at least check the field is one that has been defined.

**Pull request merged**